package/boot/uboot-sunxi/Makefile

```diff
@@ -262,6 +276,7 @@ define U-Boot/pine64_plus
   BUILD_DEVICES:=pine64_pine64-plus
   DEPENDS:=+PACKAGE_u-boot-pine64_plus:arm-trusted-firmware-sunxi
```

Comment:
don't you have to change this dependency too?

Reply:
The ATF dependencies in the second patch are adjusted to arm-trusted-firmware-sunxi-a64, when the respective package is created (when the ATF build is moved to the common code).